src/hyp3_isce2/burst.py

```diff
@@ -360,35 +360,51 @@ def get_burst_params(scene_name: str) -> BurstParams:
     )
 
 
-def validate_bursts(reference_scene: str, secondary_scene: str) -> None:
-    """Check whether the reference and secondary bursts are valid.
+def validate_bursts(reference: Union[str, Iterable[str]], secondary: Union[str, Iterable[str]]) -> None:
+    """Check whether the reference and secondary bursts are a valid sets.
```

Comment:
I think it was fine before, and the new version is grammatically incorrect :)
```suggestion
    """Check whether the reference and secondary bursts are valid.
```